exercise1/main.go

```diff
@@ -21,6 +28,13 @@ func main() {
 	// - Make a Dial -> Call the MakeDial function
 	// - Send a text message -> call the MakeText function
 
+	nokia := Mobile{
+		Brand: "Nokia",
+		Model: "N73",
+	}
+
+	MakeDial(nokia, "0822xxxxx")
```

Comment:
Since every `Mobile` should be able to make a `dial` and `send text`, we must implement the function in the line 10 above. 

So with that implementation, we should be able to call function `MakeDial` and `MakeText`